.github/workflows/automatic-release.yml

```diff
@@ -55,3 +55,13 @@ jobs:
           "SIGNING_SECRET_KEY": ${{ secrets.SIGNING_SECRET_KEY }}
           "GIT_AUTHOR_NAME": ${{ secrets.GIT_AUTHOR_NAME }}
           "GIT_AUTHOR_EMAIL": ${{ secrets.GIT_AUTHOR_EMAIL }}
+
+      - name: "Create new milestones"
+        uses: "./"
+        with:
+          command-name: "laminas:automatic-releases:create-milestones"
```

Comment:
Discussed this elsewhere with @weierophinney - unsure if we should pack this in a separate step or not 🤔

Reply:
I'm going back and forth on this.

Having it as a separate step makes it opt-in, so users can decided if this is something they want in their workflow.

On the flip side, as you mention elsewhere, adding stuff like this means we end up having to go back to repos where we've already enabled the workflow to add this step, which is time consuming and a bit of a pain.

One option might be to define a default meta-command that defines the general Laminas workflow, and internally calls each of the various commands, in order. This way as we add to the default workflow, no changes are necessary.

The one problem with this approach would be that we would either need to differentiate between the standard and the admin token in the `Variables` implementation, or we'd have to decide it's okay to use the admin token for any API calls.

But as we continue to add features, this is going to come up again, so we definitely need to address it somehow.

Reply:
If you have this in a separate step, users can configure the features they want, or do you mean merge this with the "Release" step?

Reply:
I guess it makes sense to have this as separate step :+1: